### PR TITLE
Inject attack proof

### PR DESCRIPTION
**Pull request by ramirez915**
sql statements are now sql injection attack proof according to the given instructions. Also did some refactoring

### CODE REVIEW

File: internal/accountstable/accountstable.go [hunk 1]
```diff
@@ -112,7 +108,8 @@ func MintAurumUpdateAccountBalanceTable(dbConnection *sql.DB, pkhash []byte, val
 
 func GetBalance(dbConnection *sql.DB, pkhash []byte) (uint64, error) {
 	// search for pkhash's balance
-	row, err := dbConnection.Query(sqlstatements.GET_BALANCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH + hex.EncodeToString(pkhash) + "\"")
+	// row, err := dbConnection.Query(sqlstatements.GET_BALANCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH + hex.EncodeToString(pkhash) + "\"")
```
Comment (kstola2) [inline internal/accountstable/accountstable.go]: Purpose of this comment?

File: internal/accountstable/accountstable.go [hunk 2]
```diff
@@ -236,16 +233,16 @@ func UpdateAccountTable(db *sql.DB, b *block.Block) error {
...
-			row, _ := db.Query(sqlQuery)
+			row, _ := db.Query(sqlstatements.GET_BALANCE_NONCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH, hex.EncodeToString(acc.accountPKH))
 			if row.Next() {
 				row.Scan(&balance, &nonce) // retrieve balance and nonce from account_balances
 				row.Close()
 
 				// update balance and nonce
-				sqlUpdate := fmt.Sprintf(sqlstatements.UPDATE_ACCOUNT_BALANCES_BY_PUB_KEY_HASH,
+				// sqlUpdate := fmt.Sprintf(sqlstatements.UPDATE_ACCOUNT_BALANCES_BY_PUB_KEY_HASH,
+				// 	acc.balance+int64(balance), acc.nonce+uint64(nonce), hex.EncodeToString(acc.accountPKH))
```
Comment (kstola2) [inline internal/accountstable/accountstable.go]: Comment?

File: internal/sqlstatements/sqlstatements.go [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
+	GET_BALANCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH       = "SELECT balance FROM account_balances WHERE public_key_hash = ?"
+	GET_NONCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH         = "SELECT nonce FROM account_balances WHERE public_key_hash = ?"
 	GET_HEIGHT_POSITION_SIZE_FROM_METADATA                  = "SELECT height, position, size FROM metadata"
 	GET_POSITION_SIZE_FROM_METADATA                         = "SELECT position, size FROM metadata"
 	GET_POSITION_SIZE_HASH_FROM_METADATA                    = "SELECT position, size, hash FROM metadata"
 	GET_HEIGHT_FROM_METADATA                                = "SELECT height FROM metadata"
-	GET_BALANCE_NONCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH = "SELECT balance, nonce FROM account_balances WHERE public_key_hash=\"%s\""
+	GET_BALANCE_NONCE_FROM_ACCOUNT_BALANCES_BY_PUB_KEY_HASH = "SELECT balance, nonce FROM account_balances WHERE public_key_hash = ?"
+	// LAST FINISED WAS BLOCKCHAIN...
+	// NOW MOVE ON TO CLIENT
```
Comment (kstola2) [inline internal/sqlstatements/sqlstatements.go]: Comments?

File: internal/accountstable/accountstable.go (hunk 1 above)
Comment (ramirez915) [reply internal/accountstable/accountstable.go]: Shoot! I always forget the small commented out code that I use as a reference. I will make sure to look over that more often

Comment (ramirez915) [reply internal/accountstable/accountstable.go]: I'll go back to how it was before refactoring my bad, didn't notice it as an issue

File: internal/accountstable/accountstable.go (hunk 2 above)
Comment (ramirez915) [reply internal/accountstable/accountstable.go]: got it

File: internal/sqlstatements/sqlstatements.go (hunk 3 above)
Comment (ramirez915) [reply internal/sqlstatements/sqlstatements.go]: got it

**Comment by ramirez915** [comment]
Completed

**Pull request merged**